### ISSUE TITLE
Suggestions for the .NET Core 2.2 Preview 1

### ISSUE THREAD

**Issue by Eilon** [aspnet/Templating#712]
_From @Amine-Smahi on Thursday, 23 August 2018 18:00:55_

Suggestion : 
1) The page1 an page2 navbar links should link to real pages and they should be about and contact, That will be helpful to demonstrate the routing, and how you link to other pages.
2) For better UI consistency the page1 and page2 navbar links should be aligned to the right (in mvc project)
3) Change the url from https://example.com/?action=Index&controller=Home to htps://example.com/Index or Home to make it SEO friendly
4) Add button bootstrap class to the counter button

_Copied from original issue: dotnet/core#1887_

**Comment by Eilon** [comment]
Issues (1) and (3) are bugs and I think are already fixed for the upcoming preview.

Issue (2) and (4) I'm not sure about so I'll let the template experts comment on those.

**Comment by mkArtakMSFT** [comment]
Thanks for contacting us, @Amine-Smahi.
Eilon is right, some of these issues are things we plan to have fixed in the upcoming preview2 release.

@Amine-Smahi, regarding your second point, that is something we've spotted too and @ryanbrandenburg has taken action item to fix it.
> 2. For better UI consistency the page1 and page2 navbar links should be aligned to the right (in mvc project)

@ryanbrandenburg, we should indeed use SEO friendly URLs.
> 3. Change the url from https://example.com/?action=Index&controller=Home to htps://example.com/Index or Home to make it SEO friendly

@ryanbrandenburg, please make sure this is the case:
> 4. Add button bootstrap class to the counter button

For the first point, I'll have to leave it up to @danroth27 to decide what we want to do there. This was something we explicitly did the way it is now. @danroth27, please advice how to handle this one:
> 1. The page1 an page2 navbar links should link to real pages and they should be about and contact

**Comment by ryanbrandenburg** [comment]
1. This one is all up to Dan.
2. Actually @mkArtakMSFT, Page1 and Page2 being aligned left was a very specific request from @danroth27. He wanted those links aligned left, and the Login,Register links aligned right (if they exist).
3. Where are you seeing the `?action=Index&controller=Home` links? I'm not seeing that anywhere when I test locally, but like @Eilon says, it may have just already gone in as a preview2 fix.
4. Already in my PR.

**Comment by ryanbrandenburg** [comment]
Blocked on @danroth27's responses.

**Comment by mkArtakMSFT** [comment]
@ryanbrandenburg it seems there are some things @danroth27 should clarify and that'll happen as part of preview3. For now, go ahead with part of the changes you have PRs for.

**Comment by ryanbrandenburg** [comment]
3 and 4 were merged into 2.2.0-preview2. Assigning to @danroth27 to decide on 1 and 2.

**Comment by danroth27** [comment]
@DamianEdwards I lean towards putting the navbar links consistently *on the left* and bringing back Home, About and Contact. This would keep some consistency with our earlier templates and reduce the amount of documentation churn. Having the Page1 and Page2 links that don't go anywhere makes the template feel broken. Thoughts?

**Comment by ryanbrandenburg** [comment]
I would really prefer to not bring back About and Contact. We just removed them. "Home" and "Privacy" seem like fine values there if we don't like "Page1" and "Page2".

**Comment by Eilon** [comment]
I like @ryanbrandenburg 's suggestion. I'd rather have fewer pages and make do with what we have. Making the links actually work seems reasonable.

**Comment by ryanbrandenburg** [comment]
To be clear, the "Page1" and "Page2" links both work, they just lead you to `/`, so it might appear that nothing happened.

**Comment by mkArtakMSFT** [comment]
To make it more explicit that the links actually work, what if we change the links and add querystring parameters for the page name `/?page=page1` and `?page=page2`. This would remove the confusion.

**Comment by Eilon** [comment]
Naah I don't like the query string, that also just looks like a bug.

@ryanbrandenburg thanks for the clarification. I think having it actually navigate to Home and Privacy would be good.

**Comment by Amine-Smahi** [comment]
![image](https://user-images.githubusercontent.com/24621701/44983192-43977f00-af70-11e8-819e-cb5e61d50838.png)
The button classe is missing in angular project

**Comment by Amine-Smahi** [comment]
@Eilon @mkArtakMSFT @danroth27

**Comment by ryanbrandenburg** [comment]
@Amine-Smahi we already [submitted a fix](https://github.com/aspnet/Templating/blob/release/2.2/src/Microsoft.DotNet.Web.Spa.ProjectTemplates/content/Angular-CSharp/ClientApp/src/app/counter/counter.component.html#L7) for that one but thanks for the feedback!

**Comment by DamianEdwards** [comment]
@ryanbrandenburg here's the issue we were talking about IRL today. Let's get the "Page 1" and "Page 2 " links changed to "Home" and "Privacy" as discussed.

**Comment by ryanbrandenburg** [comment]
Done in #807.
